pkg/config/templates/config.toml

```diff
@@ -184,12 +184,12 @@ enroll_enabled = true
 verify_enabled = true
 
 # Configure Multi-factor-authentication via Phone Messaging
-# [auth.mfa.phone]
-# enroll_enabled = true
-# verify_enabled = true
-# otp_length = 6
-# template = "Your code is {{ `{{ .Code }}` }} ."
-# max_frequency = "10s"
+[auth.mfa.phone]
+enroll_enabled = false
+verify_enabled = false
+otp_length = 6
+template = "Your code is {{ `{{ .Code }}` }}"
+max_frequency = "5s"
```

Comment:
**question**

Will it cause any issue if it stay commented instead of being explicitly set to false ? Wondering about retro-compatibility with existing config.

Reply:
One problem is if user sets `verify_enabled = true` without uncommenting the other fields, the update code would sync zero values to the api. I think that's rarely desired because it would override platform defaults like `otp_length`.

We can of course handle zero values in code, by treating it as null and not sending to api. But 0 is actually a valid config value according to auth team.

So I think it's cleaner to set non-zero default values for these fields to match our platform. If we want to support nullable fields in the future, we can also change them into pointers.